Dockerfile

```diff
@@ -15,19 +17,18 @@ RUN \
     yarn && \
   echo "**** install runtime packages ****" && \
   apk add --no-cache \
-    curl \
-    php8-ctype \
-    php8-curl \
-    php8-gd \
-    php8-iconv \
-    php8-intl \
-    php8-json \
-    php8-ldap \
-    php8-pdo \
-    php8-pdo_sqlite \
-    php8-phar \
-    php8-tokenizer \
-    php8-zip && \
+    php81-ctype \
+    php81-curl \
+    php81-gd \
+    php81-iconv \
+    php81-intl \
+    php81-json \
```

Comment:
json is part of the main php81 package now

Reply:
Resolved in https://github.com/linuxserver/docker-grocy/pull/62/commits/b6471985c11be934e027f692e56e8ee8478b17dc